lib/sql.js

```diff
@@ -382,6 +382,11 @@ SQLConnector.prototype.column = function(model, property) {
       return columnName;
     }
   }
+
+  // Check if name attribute provided for column name
+  if (prop && prop.name) {
+    return prop.name;
+  }
```

Comment:
IMO, connector-specific `columnName` should take precedence over `prop.name`. Please add a test to verify.

Testcase:

```
@property({
  name: 'firstName',
  mysql: {
    columnName: 'FIRST_NAME',
  }
})
```

Expected outcome:

- column name is `FIRST_NAME` on MySQL
- column name is `firstName` everywhere else (MongoDB, PostgreSQL, etc.)

Reply:
Please disregard this comment, I see that the behavior is correct and covered by a new unit test too.